### PR TITLE
Fixes in modify

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: jenkins-tools/yast-jenkins-modify.rb [hunk 1]
```diff
@@ -17,17 +17,37 @@
...
 
+# modules that do not follow yast-{mod} convention
+SPECIAL_MOD_NAMES = [
+  "skelcd-control-SLES",
+  "skelcd-control-SLED",
+  "skelcd-control-openSUSE",
+  "skelcd-control-openSUSE-LangAddOn",
+  "skelcd-control-SLES-for-VMware",
+  "y2r"
+]
```
Comment (mvidner) [inline jenkins-tools/yast-jenkins-modify.rb]: Having a hardcoded list looks problematic. Can you instead always use the git repo name?

Comment (jreidinger) [inline jenkins-tools/yast-jenkins-modify.rb]: ugh? in current way no. As jenkins jobs have different format then github one, as on jenkins we share namespece with rest of jobs, so we need yast prefix there.
Hardcoded list is currently the easiest and quicker solution for me.

File: jenkins-tools/yast-jenkins-modify.rb [hunk 2]
```diff
@@ -17,17 +17,37 @@
...
+
 ARGV.each do |mod|
```
Comment (mvidner) [inline jenkins-tools/yast-jenkins-modify.rb]: > All jobs on Jenkins have the name "yast-$MODULE-$BRANCH", but ARGV gets just a list of $MODULE

**Comment by mvidner** [comment]
Please add the clarifying comment, then LGTM.

**Pull request merged**